api/scpca_portal/test/management/commands/test_load_data.py

```diff
@@ -386,34 +395,36 @@ def test_single_cell_metadata(self):
         sample_metadata_keys = sample_metadata_lines[0].split(common.TAB)
         self.assertEqual(sample_metadata_keys, expected_keys)
 
-        # There are 4 files for each sample, plus a README.md
-        # and a single_cell_metadata.tsv file.
-        # 6 = 1 * 4 + 2
-        self.assertEqual(len(project_zip.namelist()), 6)
+        # There are 4 files for each sample, plus a README.md,
+        # a single_cell_metadata.tsv file, and 2 bulk_ files.
+        # 8 = 1 * 4 + 4
```

Comment:
```suggestion
        # There are 4 files for each sample, plus a README.md,
        # a single_cell_metadata.tsv file, and 2 bulk_ files.
        # A project with one sample should contain 8 files:
        # ex: 
        # ├── README.md
        # ├── SCPCS000001
        # │   ├── SCPCL000001_filtered.rds
        # │   ├── SCPCL000001_processed.rds
        # │   ├── SCPCL000001_qc.html
        # │   └── SCPCL000001_unfiltered.rds
        # ├── bulk_metadata.tsv
        # ├── bulk_quant.tsv
        # └── single_cell_metadata.tsv
```

I was having difficulty understanding this, I think we could either rework the wording or just add in an example for clarity.

Reply:
That's a nice visualization example! I added similar description to all modality/data outputs.